modules/ravestate_conio/__init__.py

```diff
@@ -1,29 +1,76 @@
+from h5py.h5p import DEFAULT
 
 from ravestate import registry
 from ravestate.constraint import s
+from ravestate.property import PropertyBase
 from ravestate.state import state
 from ravestate.receptor import receptor
+from ravestate.wrappers import ContextWrapper
+
 import ravestate_rawio
+import ravestate_interloc
+from ravestate_verbaliser.verbaliser import get_phrase_list
+import ravestate_phrases_basic_en
+import ravestate_ontology
+
+from scientio.ontology.node import Node
+from scientio.session import Session
+from scientio.ontology.ontology import Ontology
 
+from reggol import get_logger
+logger = get_logger(__name__)
 
-@state(read="rawio:in")
-def console_shutdown(ctx):
-    rawin = ctx["rawio:in"]
-    if "bye" in rawin:
-        ctx.shutdown()
+DEFAULT_INTERLOC_ID = "terminal_user"
 
 
-@state(triggers=s(":startup"))
-def console_input(ctx):
+@state(cond=s(":startup"), read="interloc:all")
```

Comment:
I generally am not a fan of shortening words - here cond is for Constraint but you need to think about it and look at the class definition. I don't like thinking when I do reviews...  
Also.. cond looks more like abbreviation for condescending, condolences, condoms - things we don't like 😆

Reply:
Actually it's for condition. But I do like condolences :+1: